staging/cse/windows/kubernetesfunc.ps1

```diff
@@ -105,6 +105,8 @@ function Write-KubeClusterConfig {
         HNSRemediator       = @{
             IntervalInMinutes = $Global:HNSRemediatorIntervalInMinutes;
         };
+        # IsSkipCleanupNetwork = $global:IsSkipCleanupNetwork;
+        IsSkipCleanupNetwork = $true;

```

Comment:
Why changing it to true?

Reply:
It is a testing code. Updated to the official one.